documentation/slate/source/includes/index.md

```diff
@@ -78,7 +89,7 @@ The arguments are the following:
 * ``to=-122.402770;37.794682``
 * ``datetime=20140118T0800``
```

Comment:
Not consistent with the given url (and also dates format 5 lines above).
I would rather move date explanations (these 2 lines) under the url, and provide a new resulting url (with a real datetime or a yymmddThhmmss one)